dpti/workflows/simulations/result_base.py

```
@@ -0,0 +1,112 @@
+from datetime import datetime
+from enum import Enum
+from pathlib import Path
+from typing import Any, Dict, List, Optional
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)